### PR TITLE
Update software engineer 2 typos

### PR DESCRIPTION
**Pull request by maysakanoni**

### CODE REVIEW

File: roles/sfia/software_engineer_2.md [hunk 1]
```diff
@@ -12,11 +12,11 @@ Contributor to digital, data and technology outcomes that improve society. They
 
 ### Autonomy
 
-Collaboratively leads planning of work, development of features and coordinating with stakeholders though continues to seek feedback and support on complex change.
+Collaboratively leads planning of work, development of features and coordinating with stakeholders through continues to seek feedback and support on complex change.
```
Comment (b-dalton) [inline roles/sfia/software_engineer_2.md:15]: "though" was correct in the current context

File: roles/sfia/software_engineer_2.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ Below are examples of behaviours and responsibilities a person in this role migh
 - Provides guidance and support to more junior members of the team in helping them identify what work to pick up
 - Able to iteratively deliver features of reasonable complexity, in known contexts
 - Able to architect and deliver simple features in new contexts
-- Uses a range of techniques and best practices to build easy to maintain solutions
+- Uses a range of techniques and best practises to build easy to maintain solutions
```
Comment (b-dalton) [inline roles/sfia/software_engineer_2.md:49]: Previous spelling was correct. "Practise" might be American spelling

File: roles/sfia/software_engineer_2.md (hunk 1 above)
Comment (maysakanoni) [reply roles/sfia/software_engineer_2.md:15]: I read this as though continuous feedback 😅

File: roles/sfia/software_engineer_2.md (hunk 2 above)
Comment (maysakanoni) [reply roles/sfia/software_engineer_2.md:49]: Yes it was the American spelling- so its incorrect 😜

**Pull request closed without merging**